rocketpy/plots/flight_plots.py

```diff
@@ -3,6 +3,13 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
+try:
+    from functools import cached_property
+except ImportError:
+    from ..tools import cached_property
+
```

Comment:
Why did you have to add these lines of code?

The `cached_property` can be directly imported on python versions 3.8 and above, no need for this try/except block. 

```suggestion
```

Reply:
Ah, that block must have sneaked in during a rebase to a (then newer) version of the develop branch. I will remove it again